src/components/heading/heading.component.js

```diff
@@ -70,6 +71,11 @@ class Heading extends React.Component {
      * Adds a separator between the title and the subheader.
      */
     separator: PropTypes.bool,
+
+    /**
+     * Pills that will be added after the title.
+     */
+    pills: PropTypes.node,
```

Comment:
why are we allowing any node here? Shouldn't we enforce that prop to accept only Pill components?

Reply:
There is no reason to restrict it just to Pill and actually many benefits, its common pattern used in ui libraries. eg:
- You can supply an array of elements,
- You can wrap multiple pills with fragment,
- You can use wrapper components like styled components made out of Pill ```styled(Pill)``` or other that basically return enhanced Pill,
- Leaves more flexibility for various business requirements in userland scenarios.